src/Styleguide/Pages/Artwork/Components/ArtworkSidebar/ArtworkSidebarCurrentBidInfo.tsx

```diff
@@ -6,16 +6,36 @@ import { Box } from "Styleguide/Elements/Box"
 import { Flex } from "Styleguide/Elements/Flex"
 
 import { ArtworkSidebarCurrentBidInfo_artwork } from "__generated__/ArtworkSidebarCurrentBidInfo_artwork.graphql"
+import { LoosingBid } from "Assets/Icons/LoosingBid"
+import { WinningBid } from "Assets/Icons/WinningBid"
 
 export interface ArtworkSidebarCurrentBidInfoProps {
   artwork: ArtworkSidebarCurrentBidInfo_artwork
+  readonly me?: {
```

Comment:
Not sure it's valid to combine those like this. But seems to not complain for now. Just using this to render static elements. Definitely need to change.